bridge/client/app/_components/ktb-modify-uniform-subscription/ktb-modify-uniform-subscription.component.ts

```diff
@@ -0,0 +1,153 @@
+import { Component } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
+import { DataService } from '../../_services/data.service';
+import { forkJoin, Observable, of } from 'rxjs';
+import { filter, map, switchMap, take, tap } from 'rxjs/operators';
+import { UniformSubscription } from '../../_models/uniform-subscription';
+import { DtFilterFieldDefaultDataSource } from '@dynatrace/barista-components/filter-field';
+import { Project } from '../../_models/project';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { DtFilterFieldDefaultDataSourceAutocomplete } from '@dynatrace/barista-components/filter-field/src/filter-field-default-data-source';
+import { EventTypes } from '../../../../shared/interfaces/event-types';
+
+@Component({
+  selector: 'ktb-modify-uniform-subscription',
+  templateUrl: './ktb-modify-uniform-subscription.component.html',
+})
+export class KtbModifyUniformSubscriptionComponent {
+  private taskControl = new FormControl();
+  private taskSuffixControl = new FormControl();
+  private isGlobalControl = new FormControl();
+  public data$: Observable<{ taskNames: string[], subscription: UniformSubscription, project: Project, integrationId: string }>;
+  public _dataSource = new DtFilterFieldDefaultDataSource();
+  public editMode = false;
+  public updating = false;
+  public subscriptionForm = new FormGroup({
+    taskPrefix: this.taskControl,
+    taskSuffix: this.taskSuffixControl,
+    isGlobal: this.isGlobalControl
+  });
+  public readonly affixes: { value: string, displayValue: string }[] = [
+    {
+      value: '>',
+      displayValue: '*'
+    },
+    {
+      value: 'triggered',
+      displayValue: 'triggered'
+    },
+    {
+      value: 'started',
+      displayValue: 'started'
+    },
+    {
+      value: 'finished',
+      displayValue: 'finished'
+    }];
+
+  constructor(private route: ActivatedRoute, private dataService: DataService, private router: Router) {
+    this.setValidation();
```

Comment:
This could be done directly when declaring the FormControls. The advantage may be that it is directly visible which validators are in use and no additional method and access is needed.